### PR TITLE
Fix/rfr 1036 map view crash

### PR DESCRIPTION
**Pull request by muthenberg**
Very small fix for a very rare possibility to crash.

### CODE REVIEW

File: RFR-App/RFR/Measurements/MeasurementsViewModel.swift [hunk 1]
```diff
@@ -148,8 +148,8 @@ class MeasurementsViewModel: ObservableObject {
             latitude: northWestCorner.coordinate.latitude - latitudeDistance/2,
             longitude: northWestCorner.coordinate.longitude + longitudeDistance/2
         )
-        let northSouthReach = southWestCorner.distance(from: northWestCorner)
-        let eastWestReach = northWestCorner.distance(from: northEastCorner)
+        let northSouthReach = min(southWestCorner.distance(from: northWestCorner), 0.1)
+        let eastWestReach = min(northWestCorner.distance(from: northEastCorner), 0.1)
```
Comment (hb0) [inline RFR-App/RFR/Measurements/MeasurementsViewModel.swift:152]: I don't know the context of this, but are you sure you want to cap the distance at 0.1 (max 0.1) and don't want to ensure a minimum distance or 0.1, which would be `max(..., 0.1)`?

**Review by hb0 — APPROVED** [review]
see comment

**Review by hb0 — CHANGES_REQUESTED** [review]
See my comment, from the task description it sounded like you want at least 0.1 not max 0.1 which you implemented here

File: RFR-App/RFR/Measurements/MeasurementsViewModel.swift (hunk 1 above)
Comment (muthenberg) [reply RFR-App/RFR/Measurements/MeasurementsViewModel.swift]: So you can see how tired I am all the time.

**Review by hb0 — APPROVED** [review]

**Review by hb0 — APPROVED** [review]

**Review by hb0 — APPROVED** [review]

**Comment by hb0** [comment]
There seems to be a bug here at Github, I approved but it still says "approval required".
Please just use your admin rights to merge, let's see if this bug reappears on another PR.

**Pull request merged**